conf/code-lists/pointer-mapping.csv

```diff
@@ -24,7 +24,7 @@
 42A.40A.118,declaration.totalNumberOfItems.exchangeRate,/customs-declare-exports/declaration/total-numbers-of-items
 42A.57B.R123,declaration.declarantDetails.details.eori,/customs-declare-exports/declaration/[URL_PATH]
 42A.56A.G005,declaration.locations.officeOfExit.officeId,/customs-declare-exports/declaration/office-of-exit
-42A.57A.R032,declaration.parties.exporterDetails.details.eori,/customs-declare-exports/declaration/exporter-details
+42A.57A.R032,declaration.parties.exporterDetails.details.eori,/customs-declare-exports/declaration/exporter-address
```

Comment:
Shouldn't EORI error point to exporter-eori-number instead?
Are there no pointers for exporter's address? I believe these should be added.

Reply:
Yes it should! :) Thank you for spotting that :)

Reply:
Resolved.